tests/TestBrowser.cpp

```diff
@@ -433,13 +433,12 @@ void TestBrowser::testSearchEntriesWithWildcardURLs()
 
     auto result = m_browserService->searchEntries(
         db, "https://github.com/login_page/second", "https://github.com/login_page/second");
-    QCOMPARE(result.length(), 6);
+    QCOMPARE(result.length(), 5);
     QCOMPARE(firstUrl(result[0]), QString("https://github.com/login_page/*"));
     QCOMPARE(firstUrl(result[1]), QString("https://github.com/*/second"));
     QCOMPARE(firstUrl(result[2]), QString("https://github.com/*"));
     QCOMPARE(firstUrl(result[3]), QString("http://github.com/*"));
     QCOMPARE(firstUrl(result[4]), QString("github.com/*"));
-    QCOMPARE(firstUrl(result[5]), QString("https://*.github.com/*"));
```

Comment:
Maybe we should add the ` "*.thub.com",` to the URL list to make sure it is not matched.

Reply:
Good idea